src/base/logger.h

```diff
@@ -6,17 +6,19 @@
 #include <QReadWriteLock>
 #include <QObject>
 
-const int MAX_LOG_MESSAGES = 1000;
+const int MAX_LOG_MESSAGES = 20000;
 
 namespace Log
 {
     enum MsgType
     {
-        NORMAL,
-        INFO,
-        WARNING,
-        CRITICAL //ERROR is defined by libtorrent and results in compiler error
+        ALL = -1,
+        NORMAL = 0x1,
+        INFO = 0x2,
+        WARNING = 0x4,
+        CRITICAL = 0x8 //ERROR is defined by libtorrent and results in compiler error
```

Comment:
I thought you were right and this was a leftover from my initial implementation. But after changing it I get these cryptic messages from msvc:
```
src\base\logger.h:19: error: C2059: syntax error: 'constant'
src\base\logger.h:19: error: C3805: 'constant': unexpected token, expected either '}' or a ','
```